Chefs/Views/LiveCookingPage.xaml

```diff
@@ -28,7 +28,8 @@
 		<DataTemplate x:Key="StepTemplate">
 			<!-- Workaround: Removed the ScrollViewer that was previously wrapping the content here: https://github.com/unoplatform/uno/issues/19917 -->
 			<utu:AutoLayout Spacing="24"
-							Padding="40"
+							Padding="{utu:Responsive Normal=40,
+													 Narrow=18}"
```

Comment:
The breakpoints for sizes that we use in Chefs are Normal and Wide. So it should be Normal=18 and Wide=40

But either way, I'm still seeing it with too much space on top:

<img width="421" alt="image" src="https://github.com/user-attachments/assets/b09bf77a-c146-41d2-a836-860ae3a73dec" />

This may be because AutoLayout doesn't handle Padding the same way as other controls, so it might be best to remove Padding on the AutoLayout and instead use Margins on the child elements.